composer.json

```diff
@@ -14,6 +18,6 @@
         }
     ],
     "autoload": {
-        "psr-4": {"DockerHostManager\\": "src/"}
```

Comment:
IMO, directory "DockerHostManager" is not needed in FS. I prefer let the autoload like it is actually